code/modules/research/designs/weapon_designs.dm

```diff
@@ -172,159 +122,165 @@
 	build_type = FABRICATOR
 	materials = list(/datum/material/iron = 350)
-	build_path = /obj/item/ammo_box/magazine/wt550m9
+	build_path = /obj/item/ammo_box/magazine/ds12g
 	category = list(DCAT_AMMO)
 	mapload_design_flags = DESIGN_FAB_SECURITY | DESIGN_FAB_OMNI
 
-/datum/design/plastikov9mm
-	name = "50-round magazine (9x19mm)"
-	desc = "A plastikov magazine."
-	id = "plastikov9mm"
+
+
+/datum/design/pulse_heavy
+	name = "Heavy Pulse Rifle power cell"
+	desc = "A heavy power pack designed for use with the Heavy Pulse Rifle."
+	id = "pulse_heavy"
 	build_type = FABRICATOR
-	materials = list(/datum/material/iron = 450)
-	build_path = /obj/item/ammo_box/magazine/plastikov9mm
+	materials = list(/datum/material/iron = 9000, /datum/material/silver = 900)
+	build_path = /obj/item/stock_parts/cell/pulse_heavy
 	category = list(DCAT_AMMO)
 	mapload_design_flags = DESIGN_FAB_SECURITY | DESIGN_FAB_OMNI
 
-/datum/design/uzim9mm
-	name = "32-round magazine (9x19mm)"
-	desc = "A UZI magazine."
-	id = "uzim9mm"
+/datum/design/javelin
+	name = "Javelin rack"
+	desc = "A set of javelins for the T15 Javelin Gun."
+	id = "javelin"
 	build_type = FABRICATOR
-	materials = list(/datum/material/iron = 350)
-	build_path = /obj/item/ammo_box/magazine/uzim9mm
+	materials = list(/datum/material/iron = 1000)
+	build_path = /obj/item/ammo_box/magazine/javelin
 	category = list(DCAT_AMMO)
 	mapload_design_flags = DESIGN_FAB_SECURITY | DESIGN_FAB_OMNI
 
-/datum/design/smgm9mm
-	name = "21-round magazine (9x19mm)"
-	desc = "A SMG magazine."
-	id = "smgm9mm"
+/datum/design/pulse
+	name = "Pulse magazine (standard)"
+	desc = "With a distinctive \"bell and stock\" design, pulse magazines can be inserted and removed from the Pulse Rifle with minimal effort and risk."
+	id = "pulse"
 	build_type = FABRICATOR
-	materials = list(/datum/material/iron = 350)
-	build_path = /obj/item/ammo_box/magazine/smgm9mm
+	materials = list(/datum/material/iron = 1000, /datum/material/silver = 100)
+	build_path = /obj/item/ammo_box/magazine/pulse
 	category = list(DCAT_AMMO)
 	mapload_design_flags = DESIGN_FAB_SECURITY | DESIGN_FAB_OMNI
 
-/datum/design/smgm45
-	name = "24-round magazine (.45 ACP)"
-	desc = "A SMG magazine."
-	id = "smgm45"
+/datum/design/a357
+	name = "Speed loader (.357)"
+	desc = "Designed to quickly reload revolvers."
+	id = "a357"
 	build_type = FABRICATOR
-	materials = list(/datum/material/iron = 350)
-	build_path = /obj/item/ammo_box/magazine/smgm45
+	materials = list(/datum/material/iron = 100)
+	build_path = /obj/item/ammo_box/a357
 	category = list(DCAT_AMMO)
 	mapload_design_flags = DESIGN_FAB_SECURITY | DESIGN_FAB_OMNI
 
-/datum/design/tommygunm45
-	name = "50-round drum magazine (.45 ACP)"
-	desc = "A tommygun magazine."
-	id = "tommygunm45"
+/datum/design/seeker
+	name = "Seeker shells"
+	desc = "High caliber armor piercing shells designed for use in the Seeker Rifle."
+	id = "seeker"
 	build_type = FABRICATOR
 	materials = list(/datum/material/iron = 400)
```

Comment:
```suggestion
	materials = list(/datum/material/iron = 2500)
```
Seeker rounds are super dangerous, two or three shots can disembowel a person. Needs to be more expensive.